### PR TITLE
[4.1] Undefined property notice in image layout (Re-do of #29513)

### PR DESCRIPTION
**Pull request by AndySDH**
### Summary of Changes

Fixes undefined property notice in some cases.

A fix made in #29513 was somehow un-done and lost in the echo, and the issue is now back. So fixing it again with the same solution as #29513

### Actual result BEFORE applying this Pull Request

If image_fulltext_caption is missing in the database, you get Undefined property notice in image layout
Notice: Undefined property: stdClass::$image_intro_caption

### Expected result AFTER applying this Pull Request

No PHP Notices

### CODE REVIEW

**Comment by richard67** [comment]
It seems one of the 2 changes from PR #29513 got lost with PR #30784 .

**Comment by AndySDH** [comment]
Yup, can you test this @richard67?

File: layouts/joomla/content/full_image.php [hunk 1]
```diff
@@ -38,7 +38,7 @@
 			 itemprop="image"
 			<?php echo $extraAttr; ?>
 	/>
-	<?php if ($images->image_fulltext_caption !== '') : ?>
+	<?php if (isset($images->image_intro_caption) && $images->image_intro_caption !== '') : ?>
```
Comment (Quy) [inline layouts/joomla/content/full_image.php]:
```suggestion
	<?php if (isset($images->image_fulltext_caption) && $images->image_fulltext_caption !== '') : ?>
```

**Comment by AndySDH** [comment]
Whoops, thanks for catching this @Quy, fixed

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 254ea9fbca2c97563bb4f1ca9247b61c4ea8294e<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/35597">issues.joomla.org/tracker/joomla-cms/35597</a>.</sub>

**Comment by Quy** [comment]
Please fix conflicts.

**Comment by AndySDH** [comment]
@Quy can you take over and do it? I'm not familiar with how to fix conflicts

**Comment by richard67** [comment]
@AndySDH @Quy I will fix the conflicts.

**Comment by richard67** [comment]
Done.

**Comment by AndySDH** [comment]
Thank you!

**Comment by richard67** [comment]
I have tested this item :white_check_mark: successfully on 72506264cda4c6ca353a0fa3c6b3bf661748f6fb<br /><br />By code review.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/35597">issues.joomla.org/tracker/joomla-cms/35597</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/35597">issues.joomla.org/tracker/joomla-cms/35597</a>.</sub>

**Comment by bembelimen** [comment]
Thx

**Pull request merged**